test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -133,9 +133,7 @@ test.requestHooks(mockEnrollAuthenticatorPassword)('should load select authentic
   await t.expect(selectFactorPage.getFactorSelectButtonDataSeByIndex(11)).eql('symantec_vip');
   await t.expect(selectFactorPage.getFactorDescriptionByIndex(11)).eql('Verify by entering a temporary code from the Symantec VIP app.');
 
-  // no signout link at enroll page
-  await t.expect(await selectFactorPage.signoutLinkExists()).notOk();
-
+  await t.expect(await selectFactorPage.signoutLinkExists()).ok();
```

Comment:
Another occurrence, not sure if this was intended..

@vincentvoong-okta - So this is the Enroll authenticator screen where we show a list of authenticators for the user to enroll into. With this change, we will have a `Back to sign in` link in the footer, is this ok?